.vib/kubeapps-dashboard/goss/kubeapps-dashboard.yaml

```
@@ -0,0 +1,39 @@
+file:
+  /opt/bitnami/nginx/conf/fastcgi_params:
+    exists: true
+    contains:
+      - /^fastcgi_param.*HTTP_PROXY.*\"\";/
+  /opt/bitnami/nginx/conf/nginx.conf:
+    exists: true
+    contains:
+      - /^\s+listen\s+.*8080;/
+  /opt/bitnami/scripts/nginx/bitnami-templates/default-https-server-block.conf:
+    exists: true
+    contains:
+      - /^\s+listen\s+.*8443\sssl;/
+  /opt/bitnami/nginx/conf/bitnami/certs:
+    exists: true
+    filetype: symlink
+    linked-to: "/certs"
+  /opt/bitnami/nginx/html:
+    exists: true
+    filetype: symlink
+    linked-to: "/app"
+  /opt/bitnami/nginx/logs/error.log:
+    exists: true
+    filetype: symlink
+    linked-to: "/dev/stderr"
+  /opt/bitnami/nginx/logs/access.log:
+    exists: true
+    filetype: symlink
+    linked-to: "/dev/stdout"
+commands:
```

Due to an issue caused by using `commands` instead of `command`, the `check-group-permissions` wasn't being rendered.
```suggestion
command:
```

After fixing it, the results are:
```
$ goss v kubeapps-dashboard                                                                                                      
.......................................F.....

Failures/Skipped:

Command: check-group-permissions: stdout: patterns not found: [/app, /certs, /docker-entrypoint-initdb.d]

Total Duration: 1.123s
Count: 45, Failed: 1, Skipped: 0
```

---

I've separated the /apps /certs and /docker-entrypoint-initdb.d paths for both actions to work.